app/konnect/configure/servicehub/manage-services.md

```diff
@@ -1,152 +1,218 @@
 ---
-title: Manage Services through ServiceHub
+title: Manage Services through Service Hub
 no_version: true
 ---
 
-## Services
-### Add a Service to the Catalog
+Through the [Service Hub](https://konnect.konghq.com/servicehub/), you can
+create and manage all Konnect Services, Service versions, and Service
+implementations in one place.
 
-1. From the left navigation menu, click **Services** to open ServiceHub.
+## Konnect Services
 
-2. Click **Add New Service**.
+Access all Konnect Service configuration through the
+![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+**Service Hub**.
 
-3. Enter a **Service Name**.
+### Add a Service to the catalog
 
-    A Service name can be any string containing letters, numbers, or characters;
-    for example, `service_name`, `Service Name`, or `Service-name`.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, click **Add New Service**.
 
-4. Enter a **Version Name**.
+1. Enter a **Service Name**.
 
-    A version name can be any string containing letters, numbers, or characters;
-    for example, `1.0.0`, `v.1`, or `version#1`. A Service can have multiple
-    versions.
+    A Service name can be any string containing letters, numbers, or characters;
+    for example, `service_name`, `Service Name`, or `Service-name`.
 
-5. (Optional) Enter a **Description**.
+1. (Optional) Enter a **Description**.
 
-6. Click **Create**.
+1. Click **Create**.
 
     A new Service is created and {{site.konnect_short_name}} automatically
     redirects to the Service's overview page.
 
 ### Update a Service
 
-1. From the left navigation menu, click **Services**.
-
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. Edit the Service name and description directly on this page: click on either
+1. Edit the Service name and description directly on this page: click on either
 element to reveal a text box, enter the new text, then click outside of the text
 box to save.
 
-### Delete a Service
+### Share a Service
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
+
+1. Click **Share Service**.
+
+1. Select a user or team to share the Service with.
+
+1. Select a role to grant to the user or team.
 
-1. From the left navigation menu, click **Services**.
+    You can share any Service that you have access to, with
+    the same role or lesser than what you have yourself. An organization admin
+    can share any Service with any role. For more information, see [
```

Comment:
While this entire paragraph is true. For helium, only users with the Admin role for each service will be allowed to share roles.

Since the Admin role is a superset of all service permissions, it is also implied that Admins are allowed to share any roles equal to or less than Admin with others. (AKA any role)

For some background context, the reason only admins can share is because we have not implemented the permission-to-share permission. It's like in Google Docs, you have the options of enable/disable whether editors can share permissions with others.

Reply:
This change will also be needed in #3781

Reply:
Updated. Leaving comment unresolved until I apply the changes to 3781 as well.